src/DynamoCore/Core/Threading/DynamoScheduler.cs

```diff
@@ -74,6 +74,18 @@ public interface IScheduler
         IEnumerable<AsyncTask> Tasks { get; }
```

Comment:
Though I agree that returning `Tasks` doesn't make much sense, but why do you think is unsafe? `DynamoScheduler`'s responsibility is to schedule and process these tasks, it isn't responsible for creating and destorying these tasks. From this perspective, `DynamoScheduler` is more concern with its task queue, but not tasks. 

BTW, `Tasks` is used in an extension method `AsyncTaskExtension.AllComplete()`.